location-services/connecting-to-a-portal.qmd

```diff
@@ -126,37 +140,52 @@ token
 ```
 
 :::{.callout-warning}
-Authorization tokens are temporary and will expire. If you encounter an invalid token error, you might need to generated a new token.
+Authorization tokens are temporary and will expire. If you encounter an invalid token error, you might need to generate a new token.
 :::
 
-We now need to set this token to the `ARCGIS_SECRET` environment variable. We do this with `set_auth_token()`. 
+We now need to set this token to the `ARCGIS_SECRET` environment variable. We do this with `set_arc_token()`. 
 
 ```r
-set_auth_token(token)
+set_arc_token(token)
 #> Token set to environment variable `ARCGIS_TOKEN`
 ```
 
 
-### Client flow
+#### Client flow
 
-Alternatively, we can authorize using the client OAuth2 flow. This will authorize the application we created and not ourselves. Because of this we cannot use the client flow to create or modify content. 
+Alternatively, we can authorize using the client OAuth2 flow. This will authorize the application we created and not ourselves. Because of this, we cannot use the client flow to create or modify content. 
```

Comment:
```suggestion
Alternatively, you can authorize using the client OAuth2 flow. This will authorize the application you created and not ourselves. Because of this, you cannot use the client flow to create or modify content. 
```